rllib/env/external_env.py

```diff
@@ -133,7 +133,7 @@ def log_action(self, episode_id, observation, action):
         episode.log_action(observation, action)
 
     @PublicAPI
-    def log_returns(self, episode_id, reward, info=None):
+    def log_returns(self, episode_id, reward, done, info=None):
```

Comment:
Should make this an optional arg, otherwise it would break backwards compatibility.

Reply:
Actually do we need it for non multiagent case?

Reply:
Don't need it for the non-multiagent case, but I thought for consistency. But if less = better then I'll get rid of it

Reply:
No scratch that, it's needed for the policy server as well.

Reply:
But indeed, optional arg sounds best. Guess false makes sense because either you give a done, or force it through end_episode.